src/bootstrap.js

```diff
@@ -481,6 +490,8 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
 
             ChatEngine.subscribeToPubNub();
         }
+
+        ChatEngine.isConnected = true;
```

Comment:
@ManuelFernando I have same comment for this as [here](https://github.com/pubnub/chat-engine/pull/396/files#r222119493)